optional_requirements.txt

```
@@ -1,3 +1,7 @@
 # SPDX-FileCopyrightText: 2022 Alec Delaney, for Adafruit Industries
 #
 # SPDX-License-Identifier: Unlicense
+
+numpy
+matplotlib
+serial
```

This should be `pyserial`:

```suggestion
pyserial
```